index.md

```diff
@@ -2,211 +2,159 @@
-  * Some more R, Shiny, and the tidyverse (Amanda and Rocio)
-  * More UNIX shell (Fotis)
-  * Comparative microbial genomics (Torsten)
-* 3:15-3:45pm: Tea Time!
-* 3:45-5:30pm - lab/lecture/instruction
-  * lions: Next steps working group
-  * tigers: [sourmash tutorial](sourmash.html) (Phillip Brooks)
-  * bears: [sourmash tutorial](sourmash.html) (Titus)
-
-Saturday -
+**Monday, July 1st, 2019:**  
+
+* **1:30-2:20pm:**[Welcome to ANGUS 2019 slides](http://bit.ly/2019_ANGUS_Welcome)
+	* **2:20-2:30pm:** BREAK
+* **2:30-3:30pm:** Room Welcomes & Introduction to Cloud Computing
+	* **3:30-3:45pm:** BREAK
+* **3:45pm-5:00pm:** Booting up a Jetstream Instance & Motivating Example with Visualizing BLAST scores
+	* **5:00pm:** DONE
+
+**Tuesday, July 2nd, 2019:**  
+
+* **9:00-10:20am:** [Introduction to the Shell](shell_intro/index.html)
+	* **10:20-10:40am:** BREAK
+* **10:40am-12:00pm:** [Introduction to the Shell](shell_intro/index.html)
+	* **12:00-1:00pm:** LUNCH
+* **1:00-2:00pm:** [Lisa K Johnson](https://twitter.com/monsterbashseq?lang=en): Intro to Modern Seuencing Practices 
+	* **2:00-2:10pm:** BREAK
+* **2:15-5:00pm:** Poster session & Ice Cream Social (with dairy-free options!)
+	* **5:00pm:** DONE
+
+
+**Wednesday, July 3rd, 2019:**
+
+* **9:00-10:20am:** Logging onto the Cloud
+	* **10:20-10:40am:** BREAK
+* **10:40am-12:00pm:** Conda
+	* **12:00-1:00pm:** LUNCH
+* **1:00-2:00pm:** [Amanda Charbonneau](https://twitter.com/procrastinomics?lang=en)Statistics & Experimental Design
+	* **2:00-2:10pm:** BREAK
+* **2:15-3:30pm:** Quality Control & Adapter Trimming of  Sequences
+	* **3:30-3:50pm:** BREAK
+* **3:50-5:00pm:** Biological Databases & Read Quantification 
+	* **5:00pm:** DONE
+	
+**Thursday, July 4th, 2019**
+
+* **9:00-10:20am:** Introduction to R & RStudio
+	* **10:20-10:40am:** BREAK
+* **10:40am-12:00pm:** Introduction to R & RStudio
+	* **BREAK FOR THE DAY - 4th of July Activities**
+
+
+**Friday, July 5th, 2019**
+
+* **9:00-10:20am:** RNAseq - Differential Expression
+	* **10:20-10:40am:** BREAK
+* **10:40am-12:00pm:** RNAseq - Visualization
+	* **12:00-1:00pm:** LUNCH
+* **1:00-2:00pm:** [Rayna Harris](https://twitter.com/raynamharris?lang=en) - RNAseq: a five course meal
+	* **2:00-2:10pm:** BREAK
+* **2:15-3:30pm:** Transcriptome Assembly
+	* **3:30-3:50pm:** BREAK
+* **3:50-5:00pm:** Transcriptome Annotation
+	* **5:00pm:** DONE
+
+
+
+## Second week: July 8-13, 2019.
+
+**Monday, July 8th, 2019**
+
+* **9:00-10:20am:** Mapping & Variant Calling
+	* **10:20-10:40am:** BREAK
+* **10:40am-12:00pm:** Shell Automation
+	* **12:00-1:00pm:** LUNCH
+* **1:00-2:00pm:** [Olga Botvinnik](https://twitter.com/olgabot?lang=en) - Single cell RNAseq
+	* **2:00-2:10pm:** BREAK
+* **2:15-3:30pm:** Transcriptome Assembly
+	* **3:30-3:50pm:** BREAK
+* **3:50-5:00pm:** Transcriptome Annotation
+	* **5:00pm:** DONE
+
+
+**Tuesday, July 9th, 2019**
+
+* **9:00-10:20am:** Snakemake
```

Comment:
the whole morning is snakemake; build your own snakemake for an RNA-seq workflow